### PR TITLE
Improve test workflow

### PR DESCRIPTION
**Pull request by lewfish**
This PR improves the test workflow so that it runs faster and makes it easier to see if the system works or not. It's sort of a poor man's integration test that should be run before merging every PR. It also explains how to run prediction by itself. This uses data from a new repo https://github.com/azavea/raster-vision-data/releases which allows us to distribute large files without including them in this repo.

### Testing
Read parts of the tutorial that changed and make sure you can follow the steps. 

Closes https://github.com/azavea/raster-vision/issues/199

### CODE REVIEW

File: src/rv2/samples/workflow-configs/object-detection/cowc-potsdam-test.json [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
     ],
     "test_projects": [
         {
-            "id": "2_13",
+            "id": "2-13",
             "raster_source": {
                 "geotiff_files": {
                     "uris": [
-                        "{raw}/isprs-potsdam/4_Ortho_RGBIR/top_potsdam_2_13_RGBIR.tif"
+                        "{rv_root}/processed-data/cowc-potsdam-test/2-13.tif"
```
Comment (lossyrob) [inline src/rv2/samples/workflow-configs/object-detection/cowc-potsdam-test.json]: Why move raw to rv_root? The separation was useful in my run against the s3 images without copying

Comment (lewfish) [reply src/rv2/samples/workflow-configs/object-detection/cowc-potsdam-test.json]: This is just for the `-test.json` workflow. This workflow uses a tiny dataset (containing the imagery and labels) in order to run faster.

Comment (lossyrob) [reply src/rv2/samples/workflow-configs/object-detection/cowc-potsdam-test.json]: Ahh gotcha, understood.

File: docs/object-detection.md [hunk 2]
```diff
@@ -69,50 +69,67 @@ This convention is convenient and intuitive because it stores the config file fo
...
 
-Next, you will need to copy and edit some config files.
-* Copy the Mobilenet [backend config files](../src/rv2/samples/backend-configs/tf-object-detection-api/) to `<RVROOT>/backend-configs/tf-object-detection-api/`.
-* Copy the [workflow config files](../src/rv2/samples/workflow-configs/object-detection/) to `<RVROOT>/workflow-configs/object-detection/`.
+* Copy the Mobilenet [model](http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_2017_11_17.tar.gz) that was pre-trained on COCO to `<RVROOT>/pretrained-models/tf-object-detection-api/ssd_mobilenet_v1_coco_2017_11_17.tar.gz`.
+* Copy the [test backend config file](../src/rv2/samples/backend-configs/tf-object-detection-api/mobilenet-test.config) to `<RVROOT>/backend-configs/tf-object-detection-api/mobilenet-test.config`.
+* Copy the [test data](https://github.com/azavea/raster-vision-data/releases/download/v0.0.1/cowc-potsdam-test.zip) and unzip it to `<RVROOT>/processed-data/cowc-potsdam-test`.
```
Comment (lossyrob) [inline docs/object-detection.md]: Seeing `__MACOSX/` and `.DS_Store` files in the zip - probably should keep the the zips clean of these.

Comment (lewfish) [reply docs/object-detection.md]: Yea, I'll exclude them next time I make a release.

**Pull request merged**